lib/mobility/backends/action_text.rb

```diff
@@ -42,14 +47,19 @@ def configure(options)
         # @!endgroup
       end
 
+      setup do |attributes, _options|
+        attributes.each do |name|
+          has_one :"rich_text_#{name}", -> { where(name: name, locale: Mobility.locale) },
+            class_name: "ActionText::RichText", as: :record, inverse_of: :record, autosave: true, dependent: :destroy
+          scope :"with_rich_text_#{name}", -> { includes("rich_text_#{name}") }
+          scope :"with_rich_text_#{name}_and_embeds", -> { includes("rich_text_#{name}": { embeds_attachments: :blob }) }
+        end
+      end
+
       # FIXME: replace:
       #   - `:translatable` with `options[:translatable]`
       #   - `:key` with `options[:key_column]`
-      class Translation < ::ActiveRecord::Base
-        self.table_name = "action_text_rich_texts"
-
-        belongs_to :record, polymorphic: true, touch: true
-
+      class Translation < ::ActionText::RichText
```

Comment:
As mentioned I wanted to do this since the beginning but in my tests it throws `uninitialized constant ActionText`. Also I have been writing those tests, even if using a non-final implementation, in the hopes of reaching a "green" state on the basis of which we can then clean up the mess.

Reply:
Ah sorry I didn't read carefully :sweat_smile: I also got that error which is why I added the `::`.

Reply:
Ah sorry I think the failures are because the tests only require activerecord, not all of rails.

I think it's fine to ignore the spec failures since this backend should really be a separate gem which can then just require actiontext and test with that loaded.